content/concept-demurrage.md

```diff
@@ -0,0 +1,119 @@
+# Demurrage
+
+**Warning:** Demurrage is a deprecated feature with no ongoing support. This page describes historical behavior of older versions of Ripple software.
+
+[Demurrage](http://en.wikipedia.org/wiki/Demurrage_%28currency%29) is a negative interest rate on assets held that represents the cost of holding those assets. To represent the demurrage on an issued currency in the XRP Ledger, you can track it using a custom [currency code](reference-currency.html#currency-codes) that indicates the demurrage rate. This effectively creates separate versions of the currency for each varying amount of demurrage. Client applications can support this by representing the demurraging currency code with an annual percentage rate alongside the currency code. For example: "XAU (-0.5%pa)".
+
+## Representing Demurraging Currency Amounts
+
+Rather than continuously update all amounts in the XRP Ledger, this approach divides amounts of interest-bearing or demurraging currency into two types of amount: "ledger values" recorded in the XRP Ledger, and "display values" shown to people. The "ledger values" represent the value of the currency at a fixed point, namely the "Ripple Epoch" of midnight January 1, 2000. The "display values" represent the amount at a later point in time (usually the current time) after calculating continuous interest or demurrage from the Ripple Epoch until that time.
+
+**Tip:** You can think of demurrage as similar to inflation, where the value of all assets affected by it decreases over time, but the ledger always holds amounts in year-2000 values. This does not reflect actual real-world inflation; demurrage is more like hypothetical inflation at a constant rate.
+
+Thus, client software must apply two conversions:
+
+- Taking display values at a given time and converting them to ledger values to be recorded.
+- Taking ledger values and converting them to a display value at a given point in time.
+
+### Calculating Demurrage
+
+The full formula for calculating demurrage on a currency is as follows:
+
+```
+D = A × ( e ^ (t ÷ τ) )
```

Comment:
Not for this PR, and especially not for a deprecated feature documentation, consider adding [katex](https://github.com/Khan/KaTeX) or [mathjax](https://www.mathjax.org/) for math typesetting